conf/app.routes

```diff
@@ -305,3 +305,29 @@ GET        /employer/have-paye-ref                      controllers.employer.pay
 POST       /employer/have-paye-ref                      controllers.employer.paye.DoYouHavePAYEReferenceController.onSubmit()
 
 GET        /corporation-tax                             controllers.ct.IndividualAddCorporationTaxController.onPageLoad()
+
+GET        /vat/registration-process                    controllers.vat.VatRegistrationProcessController.onPageLoad()
+
+GET        /vat/eligibility                             controllers.vat.VatEligibilityCheckController.onPageLoad()
+
+GET        /vat/vat-exception                           controllers.vat.VatRegistrationExceptionController.onPageLoad()
```

Comment:
Could we change _vat-exception_ to _has-vat-exception_ just to keep the routes consistent, please?

Reply:
done